bin/hdfcoinc/pycbc_multiifo_dtphase

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
+doing a simple monte-carlo
+
+Output is the relative SNR, time, and phase as compared to a reference
+IFO (the first one provided). 
+
+To get the noise rate this should be scaled by the local sensitivity value
+and by the SNR of the signal in the reference detector 
+(see pycbc/events/stat.py)
+"""
+import argparse, h5py, numpy.random, pycbc.detector, logging, multiprocessing
+from numpy.random import normal, uniform, power
+from scipy.stats import norm
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--ifos', nargs='+',
+                    help="The ifos to generate a histogram for")
+parser.add_argument('--sample-size', type=int, 
+                    help="Approximant number of independent samples to draw for the distribution")
+parser.add_argument('--snr-ratio', type=float, 
+                    help="SNR ratio permitted between reference ifo and all others")
```

Comment:
I don't understand this. Maximum? minimum?

Reply:
It's both i.e. you give 4, it allows 0.25 -> 4. Clarified with an example in the help string.